projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.ts

```diff
@@ -139,14 +132,21 @@ export class ItemCounterComponent
     }
   }
 
-  onBlur(event: FocusEvent) {
+  onInput(event: Event): void {
```

Comment:
When is this used?

Reply:
@LTiger14 Removed it! It was kept before when I merged develop to this branch